Sources/Services/ContainerSandboxService/SandboxService.swift

```diff
@@ -759,6 +778,221 @@ public actor SandboxService {
         }
     }
 
+    /// Checks if an EXT4 volume has never been mounted (is brand new).
+    /// Uses the EXT4 superblock mountCount field:
+    /// - mountCount == 0: Never mounted - SAFE to change ownership (brand new volume)
+    /// - mountCount > 0: Already been mounted/used - DO NOT change ownership (data may exist)
+    ///
+    /// This function is called BEFORE the volume is mounted, so new volumes will have mountCount=0.
+    private func isFirstMount(volumeImagePath: SystemPackage.FilePath) -> Bool {
```

Comment:
@dcantah Thinking about this in the context of storage enhancements that use the ext4 library.

Say we have a `container volume create --from path` command that creates and populates a filesystem in a single go. We'd need that command to know about this test and set the mount count to nonzero.

Are there other use cases where we want to setup uid/gid on a volume in instances other than this specific case (first mount, empty volume)?

For the sake of argument, what are our alternatives? Could the guest agent could perform this setup after mounting volumes and before starting the workload?

Also want to call out #729 which I don't see us doing in the immediate future but could have some interaction with this - if the volume is empty/first mount, what action(s) do we take and where?